### PR TITLE
resolves #58 create user-list

### LINKED ISSUE #58: [front] Create user-list
Usage :
```html
<user-list v-bind:users="users"></user-list>

<script>
//..
users = {
   {
     "_id": {
       "$oid": "607ae876f59d21d5a50ac3c9"
     },
     "pseudo": "mon pseudo",
     "user": "default",
     "roomsId": []
  },
  {
     "_id": {
       "$oid": "607ae876f59d21d5a50ac3c9"
     },
     "pseudo": "mon pseudo",
     "user": "default",
     "roomsId": []
  }
}
//..
</script>
```

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
**fix #58**

### CODE REVIEW

File: user-list.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/locale/fr.min.js"></script>
+	<meta charset="utf-8">
+</head>
+<body>
+	<div id="app">
+		<user-list v-bind:user-list="users"></user-list>
+	</div>
+	<script type="text/javascript">
+		const main = {
+			data(){
```
Comment (Draeggon) [inline user-list.html:16]: add name :main

File: user-list.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js "></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/locale/fr.min.js"></script>
+	<meta charset="utf-8">
+</head>
+<body>
+	<div id="app">
+		<user-list v-bind:user-list="users"></user-list>
+	</div>
+	<script type="text/javascript">
+		const main = {
```
Comment (Draeggon) [inline user-list.html:15]: add name:main

**Review by Draeggon — APPROVED** [review]

File: user-list.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
Comment (Benvii) [inline user-list.html]: File should be in front folder.

File: user-list.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                        }
+                    ]
+				}
+
+			}	
+
+		};
+		const userList = {
+			name: 'user-list',
+			props: ['user-list'],
```
Comment (Benvii) [inline user-list.html]: Needs to be in camelCase, see [vueJS Props documentation camelcase vs kebab case](https://v3.vuejs.org/guide/component-props.html#prop-casing-camelcase-vs-kebab-case)

**Review by Benvii — CHANGES_REQUESTED** [review]

File: user-list.html (hunk 3 above)
Comment (Benvii) [reply user-list.html]: Done here  : 31c247626185b7304621e40395a7eb71ab776025

File: user-list.html (hunk 4 above)
Comment (yannis-mlgrn) [reply user-list.html]: Done here : 31c247626185b7304621e40395a7eb71ab776025

File: front/user-list.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+		};
+		const userList = {
+			name: 'user-list',
+			props: ['userList'],
+			methods: {
+			},
+			template: `
+			<ul>
+			 <li v-for="user in userList" :key="user.id"> {{user.pseudo}} </li>
```
Comment (Benvii) [inline front/user-list.html:49]: v-for uses `user.id` as a key but this field doesn't exists on a user, you need to change it to `user._id.$oid`

File: front/user-list.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+				}
+
+			}	
+
+		};
+		const userList = {
+			name: 'user-list',
+			props: ['userList'],
+			methods: {
+			},
```
Comment (Benvii) [inline front/user-list.html:46]: You can remove `methods` field as it's empty

File: front/user-list.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+                        },
+                        {
+                            "_id": {
+                            "$oid": "607ae876f59d21d5a50ac3c9"
+                            },
+                            "pseudo": "mon pseudo2",
+                            "user": "default",
+                            "roomsId": []
+                        }
+                    ]
```
Comment (Benvii) [inline front/user-list.html:36]: Indentation sucks here (for users lines they should be upper than the return indentation) ^^ Maybe you can improve it

File: front/user-list.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/locale/fr.min.js"></script>
+	<meta charset="utf-8">
+</head>
+<body>
+	<div id="app">
+		<user-list v-bind:user-list="users"></user-list>
+	</div>
+	<script type="text/javascript">
+		const main = {
+            name:'main',
```
Comment (Benvii) [inline front/user-list.html:16]: Wrong indentation for name field

**Review by Benvii — CHANGES_REQUESTED** [review]

**Review by Benvii — APPROVED** [review]

**Pull request merged**